Versioning in WordPress/Rebasing Intro.md

```diff
@@ -0,0 +1,75 @@
+The following is a quick rebasing how to and when tutorial.
+
+__Scenario__: You've made a feature (`feat1`) and you need this feature for some other feature (`feat2`), so that you can continue your work. You can create a new branch from the feature branch and continue developing. During your development time, you've made a PR for the original feature.
+The PR has been approved and you merge it to the `staging` branch.
+
+Now you can rebase the new feature to the `staging` branch.
+
+While on the feature branch (`feat2`) commit all that you've developed so far. Check out the `staging` branch
+
+```bash
+git checkout staging
+```
+
+Pull all the changes to your local `staging` branch.
+
+```bash
+git pull origin staging
+```
+
+This will depend on your git settings (you might not need to define the `origin` part).
+
+After that, checkout the `feat2` branch
+
+```bash
+git checkout feat2
+```
+
+Now you can do an interactive rebase
+
+```bash
+git rebase -i origin/staging
+```
+
+You should see a commit message in the editor that you've set in the `.gitconfig` settings.
+
+You can quit (in nano it's cmd x, and in vim it's ctrl q if I'm not mistaken).
+
+Now the interactive rebase process has begun. If you have a conflict it will tell you which file is in the conflicted state. Fix the conflict and write
+
+```bash
+git add .
+```
+
+This is one of the rare cases you'll want to use the git add all command. Usually, __NEVER__ add all in one commit unless you have a small change on one feature. Otherwise, you are losing the granularity and breaking the single responsibility principle (SRP) of your commits. Remember, commits are telling a story. You should be able to read the commit history and understand what the developer was trying to do.
+
+⚠️ __DO NOT COMMIT!!!!!!__ So that means __don't type__ `git commit` command.
+
+When you've fixed all the conflicts and added the fixed files, type
```

Comment:
```suggestion
When all the conflicts are resolved and the fixed files added, type
```

Reply:
You have a passive voice in the sentence, which makes it harder to read 😉 

> When all the conflicts are resolved (by zombies) and the fixed files added (by zombies) type

https://communicatehealth.com/wehearthealthliteracy/use-zombies-to-fight-the-passive-voice/

😄